### ISSUE TITLE
Unable to login

### ISSUE THREAD

**Issue by robmadole** [cloudsmith-io/cloudsmith-cli#21]
I am issuing the following command as documented:

```
cloudsmith login
```

And it gives me the help menu.

If I add `-l` it gets a little further:

```
$ cloudsmith login -l myemail@mydomain.com
Password: 
Repeat for confirmation: 
Usage: cloudsmith login [OPTIONS] COMMAND [ARGS]...
Try "cloudsmith login -h" for help.

Error: Missing command.
```

What am I doing wrong?

**Comment by lskillen** [comment]
Hey @robmadole! It looks like there's a mistake in the code that sets up the aliases for the login command. We'll have a look for you and get it fixed for the next release, but in the meantime you should be able to use `cloudsmith token` as an alternative (just tried it here as well). :-)

**Comment by robmadole** [comment]
Thanks @lskillen I'll use that one instead :)

**Comment by lskillen** [comment]
Released a fix in `0.17.1` which you can install immediately: `pip install --upgrade cloudsmith-cli`.

Thanks for reporting the issue!

**Comment by lskillen** [comment]
@robmadole Added you to the new [thanks section](https://github.com/cloudsmith-io/cloudsmith-cli/blob/master/CONTRIBUTORS.md). :-)
